src/main/java/com/laytonsmith/core/constructs/NativeTypeList.java

```diff
@@ -129,7 +269,25 @@ public static String resolveType(String simpleName) {
 				return cl;
 			}
 		}
-		throw new ClassNotFoundException("Could not find the runner for interface of type " + methodscriptType);
+		throw new ClassNotFoundException("Could not find the runner for interface of type " + fqcn);
+	}
+
+	/**
+	 * For documentation and other purposes, it is useful to have a default, invalid instance of the underlying type.
+	 * For enums and non-enums, the process of instantiating the value is different (MEnums are actually valid as is).
+	 *
+	 * This method abstracts away the difference, and you can call this instead.
+	 * @param fqcn
+	 * @return
+	 * @throws java.lang.ClassNotFoundException
+	 */
+	public static Mixed getInvalidInstanceForUse(FullyQualifiedClassName fqcn) throws ClassNotFoundException {
```

Comment:
TODO: It may be useful for some future types to provide their own implementation of this method. We should check for the existence of a particularly named static method within the class that provides a default instance, and we first check for that, and if it exists, call that instead of instantiateUnsafe.

Reply:
Fixed